docs/app/docs/navigation/sidebar/page.mdx

```diff
@@ -0,0 +1,452 @@
+                children: [
+                  FBreadcrumbItem(onPress: () {}, child: const Text('Forui')),
+                  FBreadcrumbItem.collapsed(
+                    menu: [
+                      FTileGroup(
+                        children: [
+                          FTile(title: const Text('Documentation'), onPress: () {}),
+                          FTile(title: const Text('Themes'), onPress: () {}),
+                        ],
+                      ),
+                    ],
+                  ),
+                  FBreadcrumbItem(onPress: () {}, child: const Text('Overview')),
+                  const FBreadcrumbItem(current: true, child: Text('Installation')),
+                ],
+              ),
+              Expanded(
+                child: Container(
+                  decoration: BoxDecoration(
+                    color: context.theme.colors.muted,
+                    borderRadius: context.theme.style.borderRadius,
+                  ),
+                ),
+              ),
+              Expanded(
+                flex: 3,
+                child: Container(
+                  decoration: BoxDecoration(
+                    color: context.theme.colors.muted,
+                    borderRadius: context.theme.style.borderRadius,
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
+      );
+      ```
+    </Tabs.Tab>
+
+</Tabs>
+
+
+## CLI
+
+To generate and customize this style:
+
+```shell copy
+dart run forui style create sidebar
+```
+
+
+## Anatomy
+
+This segment describes the anatomy of the various elements of a sidebar.
+
+### `FSidebar`
+
+The widget that provides an opinionated layout on the side of the screen. The widget has the following sections:
+
+| Name    | Parameter      | Position   | Optional |
+|:--------|:---------------|:-----------|:---------|
+| header  | header         | sticky     | ✅       |
+| content | child/children | scrollable | ❌       |
+| footer  | footer         | sticky     | ✅       |
+
+### `FSidebarGroup`
+
+A widget that is used to group several `FSidebarItem`s.
+
+### `FSidebarItem`
+
+Represents an item on the sidebar. May be nested in `FSidebarItem.children` to create nested items.
+
+
+## Usage
+
+### `FSidebar(...)`
+
+```dart copy
+FSidebar(
+  width: 250,
+  header: const Text('Header'),
+  footer: const Text('Footer'),
+  children: [],
+);
+```
+
+### `FSidebar.builder(...)`
+
+```dart copy
+FSidebar.builder(
+  width: 250,
+  header: const Text('Header'),
+  footer: const Text('Footer'),
+  itemCount: 5,
+  itemBuilder: (context, index) => FSidebarItem(),
+);
+```
+
+### `FSidebar.raw(...)`
+
+```dart copy
+FSidebar.raw(
+  width: 250,
+  header: const Text('Header'),
+  footer: const Text('Footer'),
+  child: SizedBox(),
+);
+```
+
+### `FSidebarGroup(...)`
+
+```dart copy
+FSidebarGroup(
+  label: const Text('Group'),
+  action: const Icon(FIcons.plus),
+  onActionPress: () {},
+  children: [],
```

Comment:
I think it will be nice to include the children's expected widgets, i.e. `FSideItem(...)`.